configurations.md

```diff
@@ -21,9 +22,9 @@ The Flink connector library for Pravega supports the Flink Streaming API, Table
 ## Common Configuration

```

Comment:
@vijay in TOC configuration section is given but content is missing? please check

Reply:
let's remove the link `configuration-elements`